embedded-postgresql/README.md

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
not sure we need this file, you may remove it, instead please add new configuration properties description to root MD file